dashboard/client/src/App.tsx

```diff
@@ -243,6 +243,10 @@ const App = () => {
                       element={<ServeReplicaDetailPage />}
                       path=":deploymentName/:replicaId"
                     />
+                    <Route
+                      element={<TaskPage />}
+                      path=":deploymentName/:replicaId/tasks/:taskId"
+                    />
```

Comment:
Can we make this a child route of `<Route
                      element={<ServeReplicaDetailPage />}
                      path=":deploymentName/:replicaId"
                    />`? 
                    
...so the breadcrumbs work correctly.
You may need to do something similar to `ActorDetailLayout`, `ActorDetailPage`, `TaskPage` in line 221.

Reply:
Updated, please have a look at the recording above